README.rst

```diff
@@ -19,28 +19,43 @@ OpenWISP 2 monitoring module (Work in progress).
 
 ------------
 
-Install Depdendencies
+Install Dependencies
 ---------------------
 
-`Install influxdb <https://docs.influxdata.com/influxdb/v1.4/introduction/installation/>`_ eg:
+We use InfluxDB to store metrics and Redis as celery broker (you can use a different
+broker if you want). The recommended way for development is running them using Docker
+so you will need to `install docker and docker-compose <https://docs.docker.com/engine/install/ubuntu/>`_
+beforehand.
+
+In case you prefer not to use Docker you can `install InfluxDB <https://docs.influxdata.com/influxdb/v1.8/introduction/install/>`_
```

Comment:
IMO, we can skip code block for installation of influxdb, since these are the same command which are mentioned in the linked documentation.

Reply:
Good. I wasn't sure about keeping it because it was there already but I will remove it and link to the official documentation. :+1: